javav2/example_code/ec2/src/main/java/com/example/ec2/MonitorInstance.java

```diff
@@ -34,20 +26,24 @@
 public class MonitorInstance {
 
     public static void main(String[] args) {
-        final String USAGE =
-                "To run this example, supply an instance id and a monitoring " +
-                        "status\n" +
-                        "Ex: MonitorInstance <instance-id> <true|false>\n";
+
+        final String USAGE = "\n" +
+                "Usage:\n" +
+                "MonitorInstance <instanceId> <monitor>\n\n" +
+                "Where:\n" +
+                "    instanceId - an instance id value that you can obtain from the AWS Console. \n\n" +
+                "    monitor - a monitoring status (true|false)";
 
         if (args.length != 2) {
             System.out.println(USAGE);
             System.exit(1);
         }
 
+        // Read the command line arguments

```

Comment:
Are these necessary? The code following is obvious. I say delete them.

Reply:
OK - I will delete these strings from V2 examples